backend/test/deal-observer.test.js

```diff
@@ -107,6 +107,36 @@ describe('deal-observer-backend', () => {
     await storeBlockEvent(data)
     assert.strictEqual(await countStoredActiveDeals(pgPool), 2n)
   })
+
+  it('deal with duplicate events', async () => {
```

Comment:
Based on the [discussion on Slack](https://space-meridian.slack.com/archives/CP50PPW2X/p1739464959149629), the events were not duplicate. What happened: a client stored two copies of the same piece in one sector. The events were for different datacap claims (the claim ID was different).

I feel it's important to capture this distinction in the test name.

How about this:

```suggestion
  it('handles one client piece stored twice in the same sector', async () => {
```

Reply:
Agreed, this deal stuff is confusing enough already, let's have our code be clear